coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverterTest.java

```diff
@@ -25,6 +25,8 @@
 // This makes it easier to generate RelNodes for testing. The input sql is
 // in Calcite sql syntax (not Hive)
 // Disabled tests are failing tests
+@Test(enabled = false,
+    description = "pending migration to hive tables and corresponding queries to use standardised CoralSqlNode and CoralRelNode representations in the translation path")
```

Comment:
Looks like some of the tests pass if they are enabled, why do we need to migrate them?